modules/API/external/external_api.js

```diff
@@ -366,6 +366,29 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return iframe.contentWindow.document.getElementById('largeVideo');
     }
 
+    /**
+     * Getter for participant specific video element in Jitsi Meet.
+     *
+     * @param {string} participantId - Participant id or "local" for local video.
+     *
+     * @returns {HTMLElement|undefined} - The requested video.
+     */
+    _getParticipantVideo(participantId) {
```

Comment:
I also get couple of "Strings must use singlequote.eslint(quotes)" ESLint errors....

Did you disabled the pre commit hook that runs the eslint and flow validation? or maybe you haven't run npm install?

Please make sure that you are using the precommit hook and validation is passing. If I merge the PR with an eslint error that means anybody who pulls the changes won't be able to commit and will have to spend the time to find and fix the error.